iroh/src/commands.rs

```diff
@@ -60,6 +61,65 @@ pub struct Cli {
 impl Cli {
     pub async fn run(self, rt: &runtime::Handle, config: &Config) -> Result<()> {
         match self.command {
+            Commands::Share {
+                hash,
+                recursive,
+                rpc_port,
+                peer,
+                addr,
+                token,
+                ticket,
+                derp_region,
+                mut out,
+                stable: in_place,
+                ..
+            } => {
+                if let Some(out) = out.as_mut() {
+                    tracing::info!("canonicalizing output path");
+                    let absolute = std::env::current_dir()?.join(&out);
+                    tracing::info!("output path is {} -> {}", out.display(), absolute.display());
+                    *out = absolute;
+                }
+                let client = make_rpc_client(rpc_port).await?;
+                let (peer, addr, token, derp_region, hash, recursive) =
+                    if let Some(ticket) = ticket.as_ref() {
+                        (
+                            ticket.peer(),
+                            ticket.addrs().to_vec(),
+                            ticket.token(),
+                            ticket.derp_region(),
+                            ticket.hash(),
+                            ticket.recursive(),
+                        )
+                    } else {
+                        (
+                            peer.unwrap(),
+                            addr,
+                            token.as_ref(),
+                            derp_region,
+                            hash.unwrap(),
+                            recursive.unwrap_or_default(),
+                        )
+                    };
+                let mut stream = client
+                    .server_streaming(ShareRequest {
+                        hash,
+                        recursive,
+                        peer,
+                        addrs: addr,
+                        derp_region,
+                        token: token.cloned(),
+                        out: out.map(|x| x.display().to_string()),
+                        in_place,
+                        force: true, // todo: use force from Commands::Share
```

Comment:
what about this todo?

Reply:
This was quite useful during development. Otherwise it would just do nothing all the time. I enabled this now, but the question is if we should have it at all. Maybe just remove the force mechanism entirely?